docs/websockets.rst

```diff
@@ -201,3 +201,43 @@ Having API keys for some services (such as etherscan) is crucial for rotki to wo
 
 - ``service``: Service for which an API key is needed.
 - ``location``: For services like etherscan that require different API keys for different locations this is the location for which the API key is needed. If a service does not require a location then this key will be missing.
+
+
+Query new history events
+=========================
+
+In addition to history from evm transactions we need to query events from exchanges that support this kind of events. The backend will emit a ws message as the following when a query start for such exchange.
+
+::
+
+    {
+        "type": "querying_events_status",
+        "data": {
+            "status": "querying_events_started",
+            "location": "kraken",
```

Comment:
Should it be location or also even type? Since location alone won't tell us much for eth withdrawals or eth block production